### ISSUE TITLE
Support for the splitkb.com kyria

### ISSUE THREAD

**Issue by elkowar** [Ciantic/qmkmapper#10]
I'd love to try to use this, but my keyboard, the kyria, seems to be missing! I don't quite know where the keyboards are found/listed here, but it would be great If the kyria (and probably other missing keyboards) where added

**Comment by elkowar** [comment]
from what I gather, the only thing that'd be necessary is to extend the list of urls the keymaps are fetched from to include the newer pages. Should be a trivial change, even for a seemingly abandoned project 👀

**Comment by Ciantic** [comment]
@elkowar it's not abandoned, it's just working :) I think all I have to do is rebuild with clean cache.

**Comment by elkowar** [comment]
would be great then! but from what I can tell you're only fetching the first 5 pages, but there are at this point a lot more than 5 pages ^^

**Comment by Ciantic** [comment]
Yeah, there was no more than five pages when I last time fetched the cache 👍 

Though, I'm fighting here with node version and npm version, which I haven't updated for few years, and things feel like broken.

I'll try to push a new version when I get this working.

**Comment by aphillippe** [comment]
owner of a Kyria here. I've just installed the vscode extension and can't find the kyria. I assume the above rebuild never happened? This extension looks awesome if I can get it working for my kb @Ciantic
